src/main.rs

```diff
@@ -41,5 +44,15 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         clue_collection.display();
     }
 
+    // If -t is set, output the token usage information
+    if args.token {
+        // Write to stderr in the format: prompt_tokens, completion_tokens, total_tokens
+        writeln!(
```

Comment:
`writeln!(std::io::stderr(), ... )` is totally valid here, but it can be simplified using `eprintln!()`